### PR TITLE
Fixes #25183 - Deprecate old repo/cvv export API

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: app/controllers/katello/api/v2/repositories_controller.rb [hunk 1]
```diff
@@ -272,7 +272,7 @@ def sync
       raise HttpErrors::BadRequest, e.message
     end
 
-    api :POST, "/repositories/:id/export", N_("Export a repository")
+    api :POST, "/repositories/:id/export", N_("Export a repository"), :deprecated => true
```
[the remark this answers is not in the document]
Comment (beav) [reply app/controllers/katello/api/v2/repositories_controller.rb]: I agree. I originally thought we'd deprecate both, but its better to leave this one around at least for now.

**Comment by chris1984** [comment]
@ehelms updated.

**Comment by beav** [comment]
[test katello]

**Comment by chris1984** [comment]
Tests are passing

**Pull request merged**